src/main/java/de/sakpaas/backend/service/SearchService.java

```diff
@@ -81,14 +81,19 @@ public SearchResultObject search(String query, CoordinateDetails coordinateDetai
 
 
   /**
-   * Creates a new SearchRequest Object. Also makes all query entrys lower case.
+   * Creates a new SearchRequest Object. Also makes all query entries lower case.
    *
    * @param query             The SearchQuery
    * @param coordinateDetails The SearchCoordinates
    * @return A new SearchRequest
+   * @throws EmptySearchQueryException Iff the query is empty
```

Comment:
typo

Reply:
Meinst du `Iff`? Wenn ja dann ist das `If and only if` :D

Reply:
oh, nevermind